components/automate-cli/cmd/chef-automate/iam.go

```diff
@@ -420,7 +421,7 @@ func runCreateTokenCmd(cmd *cobra.Command, args []string) error {
 	tokenResp, err := apiClient.TokensV2Client().CreateToken(ctx, &policies_req.CreateTokenReq{
 		Id:     id,
 		Name:   name,
-		Active: true,
+		Active: &wrappers.BoolValue{Value: true},
```

Comment:
[nit] it defaults to _active_, so, we might as well go without this line
```suggestion
```
but being more explicit is fine. LGTM.

Reply:
Smart.